tools/bumper/bumper.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+var Logger *log.Logger
+
+type inputParams struct {
+	componentsConfigPath string
+	gitToken string
+}
+
+func main() {
+	var inputArgs inputParams
+	Logger = initBumper(&inputArgs)
```

Comment:
The `initBumper` means nothing and you are not going to re-use this, don't add not needed indirections, just put here the logger initialization and call the initFlags

Reply:
Done